scripts/exregional_run_analysis.sh

```diff
@@ -247,6 +255,7 @@ case $MACHINE in
   done
 
   if [ $foundens ]; then
+    enkfcstname=gdas
```

Comment:
Why use a variable here? And why code this? Also, if you decide to keep it, should it be `enkffcstname`?

Reply:
This is to fix a crash when no ensemble forecast available. The enkfcstname needs a value to avoid crash.

Reply:
This variable is used many time before this line. Please check the code before this line.

Reply:
In all other instances above this line, it is used in a MUCH different way than it is used here. Previously in this script it sets the full templated file name, so you're actively re-setting it here when I don't think that it is needed since you are only using it in the line 259. This additional variable adds nothing and can lead to more complexity in the code.

Reply:
Now I see this is not an good fixing to the problem we have when GFS ensemble forecast is missing. It induced another bug when the GFS ensemble is available.
I have changed the way to fix the GFS ensemble forecast missing problem. Please review.